### DISCUSSION TITLE
How to detect if a file is imported by `new URL(...)`?

### DISCUSSION THREAD

**Discussion by latin-1** [webpack/webpack#12329] [Q&A] [answered]
I set up the svgr loader for all svg files, and it works fine in combination with file-loader, as the docs said.

https://react-svgr.com/docs/webpack/#using-with-url-loader-or-file-loader

However, since it will create a wrapper module for every svg file, it doesn't work together with `new URL(...)`.

I hope there will be a new property to detect the import type of that file. (e.g. `issuerType: "URL"`)

**Comment by sokra** [comment, marked as answer]
`dependencyType: "url"` should do that in the rule

**Reply by latin-1** [reply]
Thanks a lot! You made my day!
